packages/tables/src/Table/Concerns/CanSortRecords.php

```diff
@@ -71,11 +71,6 @@ public function getDefaultSortDirection(): ?string
         return $direction;
     }
 
-    public function getDefaultSortQuery(): ?Closure
```

Comment:
For BC, we can keep this method and inside `return ($this->defaultSort instanceof Closure) ? $this->defaultSort : null`

Reply:
Please mark it as `@deprecated` with a replacement like the other places we do that